python/lib/dependabot/python/file_updater/pip_compile_file_updater.rb

```diff
@@ -220,7 +220,7 @@ def write_updated_dependency_files
         end
 
         def install_required_python
-          return if run_command("pyenv versions").include?("#{python_version}\n")
+          return if run_command("pyenv versions").include?(" #{python_version}")
```

Comment:
Is the preceding space important here? If so, what do you think about using `"\ #{`? I'm able to notice it here because of diff highlighting, but I'd be much less likely to notice it when I need to.

Reply:
Yes the space is important, I've applied the suggested change.